api-explorer/index.html

```diff
@@ -76,13 +76,13 @@
 <script src="./swagger-ui-standalone-preset.js"> </script>
 <script>
 window.onload = function() {
-  
+
   // Build a system
   // https://github.com/swagger-api/swagger-ui/blob/master/docs/usage/configuration.md
   const ui = SwaggerUIBundle({
-    // Default to petstore. LoopBack 4 applications will override it by setting 
+    // Default to petstore. LoopBack 4 applications will override it by setting
     // the url parameter, such as https://loopback.io/api-explorer?url=<swagger-spec-url>
-    url: "http://petstore.swagger.io/v2/swagger.json",
+    url: "https://petstore.swagger.io/v2/swagger.json",
```

Comment:
seems like the content for both links are the same? Any reason we want to change it?

Reply:
It’s to fix the mixed-content issue when accessing via https://loopback.io/api-explorer/